workflows/batch_workflow.py

```diff
@@ -173,10 +176,12 @@ def main(  # pylint: disable=too-many-arguments,too-many-locals,too-many-stateme
     samples_df, samples_csv_path, pre_combiner_jobs = _add_pre_combiner_jobs(
         b=b,
         input_gvcfs_bucket=f'gs://cpg-{callset_name}-{input_bucket_suffix}/gvcf',
+        input_metadata_bucket=f'gs://cpg-{callset_name}-{input_bucket_suffix}-metadata'
```

Comment:
Arguably `input_bucket_suffix` is now not only a suffix anymore -- should we rename `input_bucket_suffix` to `namespace`?

Reply:
Definitely!